jujugui/static/gui/src/app/components/generic-button/test-generic-button.js

```diff
@@ -36,7 +36,9 @@ describe('GenericButton', function() {
     var output = jsTestUtils.shallowRender(
         <juju.components.GenericButton
           action={callbackStub} />);
-    output.props.onClick();
+    output.props.onClick({
+      stopPropagation: sinon.stub()
```

Comment:
Why stub instead of spy? My understanding is that (in the Sinon world) a stub is a spy with specified behavior (return this value, throw this error, etc.). We don't need to specify any behavior here, so a spy seems like a better fit. That said, this is test bike shedding.